### PR TITLE
assertions evalPbc

### PR DESCRIPTION
**Pull request by grewered**

### CODE REVIEW

**Comment by grewered** [comment]
the checks from the pull request didn't run automaticaly, how can I run them manually?

File: R/evalPbc.r [hunk 1]
```diff
@@ -2,6 +2,10 @@ evalPbc <- function(pbcs, mistypes = c("mnr", "mbd", "mir", "mbi"),
                     minPbcAtt = .05, maxPbcDis = .005, maxPbcMis = .07) {
   cli_setting()
 
+  checkmate::assert_data_frame(pbcs)
+  checkmate::assert_vector(mistypes)
```
[the remark this answers is not in the document]
Comment (sachseka) [reply R/evalPbc.r:6]: you are completely right! thus, this check is fine

**Review by sachseka — APPROVED** [review]
LGTM!

File: R/evalPbc.r (hunk 1 above)
[the remark this answers is not in the document]
Comment (grewered) [reply R/evalPbc.r:6]: shall I change the man page, so that it says character or numeric?

**Review by sachseka — APPROVED** [review]

**Pull request merged**